### DISCUSSION TITLE
How to use the PMTiles?

### DISCUSSION THREAD

**Discussion by naimkhrof** [maplibre/maplibre-react-native#574] [Q&A] [answered]
Hello, i followed this documentation https://docs.protomaps.com/pmtiles/maplibre#installation and Im asking if the pmtiles protocol is by default included in maplibre-react-native. And i i can add manually one thank you

official documentation:
https://maplibre.org/maplibre-gl-js/docs/API/functions/addProtocol/

**Comment by KiwiKilian** [comment, marked as answer]
Please have a look at issue #28, PMTiles aren't supported yet by MapLibre Native and therefore also not supported through MapLibre React Native.
